core/src/trezor/ui/model/t1/layout.py

```diff
@@ -0,0 +1,18 @@
+from trezor import ui
+
+from .confirm import Confirm
+from .text import Text
+
+if False:
+    from typing import Dict, Callable
+
+
+def layout_confirm_ping(content: Dict[str, str]) -> Confirm:
+    text = Text("header")
+    text.normal("ping?")
+    return Confirm(text, confirm="CONFIRM", cancel="CANCEL")
+
+
+LAYOUTS = {
```

Comment:
this seems superfluous, you could as well use `getattr(layout, "confirm_ping")` in the lookup layout function